From c62cc5a9642a9da893f28804a6e06496a1878521 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Fri, 26 Jun 2020 17:26:30 +0200
Subject: [PATCH] [mediacontroller] Use half the samples for better performance
 with swrast

With swrast (llvmpipe) the artwork blurring is enabled, which takes an eternity
(up to ~15s) to compile the shader. Improve that by only using half the
samples with half the radius, which looks nearly identical.
---
 .../mediacontroller/contents/ui/ExpandedRepresentation.qml    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml b/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml
index 5653bcc38..2f89d33cc 100644
--- a/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml
+++ b/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml
@@ -165,9 +165,9 @@ Item {
                     layer.effect: GaussianBlur {
                         cached: true
 
-                        radius: 256
+                        radius: 128
                         deviation: 12
-                        samples: 129
+                        samples: 63
 
                         transparentBorder: false
                     }
-- 
2.25.1

